wordclock/settings.h

```diff
@@ -28,14 +28,24 @@ typedef enum {
     HARDWARE_11_11 = 0,
     HARDWARE_13_13 = 1,
     HARDWARE_9_8 = 2,
+    HARDWARE_13_13_V2 = 3,
+    NR_OF_HARDWARE = HARDWARE_13_13_V2,
 } EHardwareType;
 
+typedef enum {
+    PIXEL_TYPE_RGB = 0,
+    PIXEL_TYPE_RGBW = 1,
+    PIXEL_TYPE_RGBNW = 2,
+    PIXEL_TYPE_RGBWW = 3,
+    NR_OF_PIXEL_TYPES = PIXEL_TYPE_RGBWW,
```

Comment:
en je kunt toch ook niet twee enums met dezelfde waarde gebruiken? of zie it nu iets over het hoofd?

Reply:
Ja dat kan wel maar idd het nummer klopt niet.